homeassistant/components/zwave_js/discovery.py

```diff
@@ -858,6 +870,17 @@ def get_config_parameter_discovery_schema(
     ),
     # cover
     # window coverings
+    ZWaveDiscoverySchema(
+        platform=Platform.COVER,
+        hint="window_covering",
+        primary_value=WINDOW_COVERING_COVER_CURRENT_VALUE_SCHEMA,
```

Comment:
Will the window covering values only get discovered with v11 of Z-Wave JS or will it be discovered already with v10?

I'm thinking about our release strategy and breaking changes. If we don't get the architecture discussion solved before the beta we may want to hold off merging here and potentially ask Z-Wave JS to also hold off releasing v11, if there are breaking changes.

Reply:
Window Covering CC is already supported in v10. Main value of pushing this forward is because of this issue: https://github.com/zwave-js/node-zwave-js/issues/5686

Reply:
Are there no breaking changes regardless if we release this with or without v11. Ie it doesn't matter if we release this now, also if v11 is applied later without further changes in core?

Reply:
that's correct - in v11, devices that have the Window Covering CC will no longer advertise the Multilevel Switch CC values, but once this change is merged that won't impact users since the device will now attach to the Window Covering CC values instead of the Multilevel Switch CC values anyway